Test/Integration/AddScriptTest.php

```diff
@@ -23,43 +23,55 @@ protected function setUp()
     /**
      * @magentoConfigFixture default/web/cookiebot/enabled 1
      * @magentoConfigFixture default/web/cookiebot/id 123-456-789
+     * @magentoConfigFixture current_store web/cookiebot/use_eu_cdn 1
      */
     public function testScriptAddedOnHomepage(): void
     {
         $this->dispatch('/');
         self::assertContains($this->script, $this->getResponse()->getBody());
+        self::assertEquals('<script id="Cookiebot" data-cfasync="false" src="https://consent.cookiebot.eu/uc.js" data-cbid=""  type="text/javascript" async></script>',
```

Comment:
I don't think we need to test this here, since we have such tests for the `ScriptGenerator`.

Reply:
Actually I remember in your previous comments you also asked me to add these tests here as well

Reply:
https://github.com/customgento/module-cookiebot-m2/pull/23/files/461bda69f89c9927171ea1daa8ac01f25e141abb#r1404128423 second part

Reply:
I asked to add the EU option. What I meant is: We should make sure that the script is inserted on the page no matter if the EU option is set or not. We don't need to test for the actual HTML as this is done by the `ScriptGeneratorTest`. But we should e.g. add `testScriptAddedOnHomepageWithEuCdn`, which tests if the script is inserted on the homepage when the EU CDN is enabled.

Reply:
Actually I think you are talking about adding an extra duplicate test, since we also tested it in another file and I think it would extra duplicate test if we again here test the script tag in the html. what's you idea? 
the steps would be like
1_Test if script tag is in the html page
2_Test if eu cdn is true and works fine and Again if the test script tag is in the html page
3_In future for each single configuation we also want to check if script tag is in html page separately?